### PR TITLE
Quality of life tweaks

### PR DESCRIPTION
**Pull request by cohan**
Just a couple of quick changes to warm up

1. Add a link to the download - this link is pointing to the current version of the tool
2. Move the configuration to the top of the file. I noticed on a couple of streams during setup it was a bit tricky to find what needed editing

Edit: Sorry, forgot to say: I've tested to confirm that it's working as it was before I tinkered with it

-- corobo

### CODE REVIEW

**Comment by wouter-veeken** [comment]
Probably not something for this PR, but consider adding a few quick screenshots to the README while you're updating it

File: twitchpopups.htm [hunk 1]
```diff
@@ -1,6 +1,35 @@
 <!DOCTYPE HTML>
 <html>
 <head>
+
+    <script type="text/javascript">
+
+        // Change Limmy to your Twitch channel
```
Comment (vandelay87) [inline twitchpopups.htm:7]: get rid of these comments. code should be clean and readable without the aid of comments

File: twitchpopups.htm [hunk 2]
```diff
@@ -1,6 +1,35 @@
 <!DOCTYPE HTML>
 <html>
 <head>
+
+    <script type="text/javascript">
+
+        // Change Limmy to your Twitch channel
+        var twitchChannel = 'Limmy';
```
Comment (vandelay87) [inline twitchpopups.htm:8]: this value is constant. change to `const TWITCH_CHANNEL = 'Limmy';`

File: twitchpopups.htm [hunk 3]
```diff
@@ -1,6 +1,35 @@
...
+        // Your alert background. Default is a vibrant green
+        var alertBg = '#00AA00';
+
+        // Hotseat background colour. Default is a firey, firey orange.
+        var hotseatBg = '#aa1100';
+
+        // The emoji that surrounds the hotseat messages.
+        var hotseatEmoji = '🔥';
+        // To open an emoji keyboard on your computer:
+        //     Windows: Windows Key + Period (.)
```
Comment (wouter-veeken) [inline twitchpopups.htm:19]: @DaftLimmy most programmers (or streamers) are not native English speakers and will be more familiar with American English

File: twitchpopups.htm [hunk 4]
```diff
@@ -1,6 +1,35 @@
...
 <html>
 <head>
+
+    <script type="text/javascript">
+
+        // Change Limmy to your Twitch channel
+        var twitchChannel = 'Limmy';
+
+        // Your alert background. Default is a vibrant green
+        var alertBg = '#00AA00';
```
Comment (vandelay87) [inline twitchpopups.htm:11]: more constants

File: twitchpopups.htm [hunk 5]
```diff
@@ -91,15 +117,15 @@ <h1 id="popuptext"></h1>
...
-                  $("#popupbox").css({"background-color":"#aa1100"}); 
-                  $("#popuptext").text("🔥 " + hotseatuser.toUpperCase() + " IS IN THE HOTSEAT 🔥");
+                  $("#popupbox").css({"background-color":hotseatBg}); 
+                  $("#popuptext").text(hotseatEmoji + " " + hotseatuser.toUpperCase() + " IS IN THE HOTSEAT " + hotseatEmoji);
                   doAnimation();
                 }
             }
                 
             if (hotseatison) {
                 if (context.username == hotseatuser){
```
Comment (vandelay87) [inline twitchpopups.htm:127]: match the type too `===`

File: twitchpopups.htm [hunk 6]
```diff
@@ -91,15 +117,15 @@ <h1 id="popuptext"></h1>
                   hotseatison = true;
                   $("#popupbox").show();
                   hotseatuser = commandName.substr(10).toLowerCase();
-                  $("#popupbox").css({"background-color":"#aa1100"}); 
-                  $("#popuptext").text("🔥 " + hotseatuser.toUpperCase() + " IS IN THE HOTSEAT 🔥");
+                  $("#popupbox").css({"background-color":hotseatBg}); 
+                  $("#popuptext").text(hotseatEmoji + " " + hotseatuser.toUpperCase() + " IS IN THE HOTSEAT " + hotseatEmoji);
```
Comment (vandelay87) [inline twitchpopups.htm:121]: better to use template literals here
`$("#popuptext").text(`${hotseatEmoji} ${hotseatuser.toUpperCase()} IS IN THE HOTSEAT ${hotseatEmoji}`);`

Comment (sirlemonhead) [reply twitchpopups.htm:121]: This is a joyful and wondrous suggestion!

Comment (AmyJoT) [reply twitchpopups.htm:121]: Preferable, rather than better I think

File: twitchpopups.htm (hunk 1 above)
Comment (AmyJoT) [reply twitchpopups.htm:7]: For a tool like this I think the comments are perfectly fine.

File: twitchpopups.htm (hunk 6 above)
Comment (limmyshagsdugs) [reply twitchpopups.htm:121]: how's it going fellas

**Comment by frshhh** [comment]
I just reviewed ALL the things on this page and as such I give Lamby my express permission to commit to them or not. You have my blessing.

No need to VIP me. :)

File: twitchpopups.htm (hunk 1 above)
Comment (wouter-veeken) [reply twitchpopups.htm:7]: It's nonsense even as a general statement, but especially in this case, considering target audience, it's just obviously untrue.

Comment (cohan) [reply twitchpopups.htm:7]: > get rid of these comments. code should be clean and readable without the aid of comments

This isn't code, it's configuration.

File: twitchpopups.htm [hunk 7]
```diff
@@ -91,15 +117,15 @@ <h1 id="popuptext"></h1>
...
+                  $("#popupbox").css({"background-color":hotseatBg}); 
+                  $("#popuptext").text(hotseatEmoji + " " + hotseatuser.toUpperCase() + " IS IN THE HOTSEAT " + hotseatEmoji);
                   doAnimation();
                 }
             }
                 
             if (hotseatison) {
                 if (context.username == hotseatuser){
-                    $("#popuptext").text("🔥 " + context['display-name'] + ": " + commandName + " 🔥");
+                    $("#popuptext").text(hotseatEmoji + " " + context['display-name'] + ": " + commandName + " " + hotseatEmoji);
```
Comment (mattgirdler) [inline twitchpopups.htm:128]: More template literals here

File: twitchpopups.htm [hunk 8]
```diff
@@ -91,15 +117,15 @@ <h1 id="popuptext"></h1>
                   hotseatison = true;
```
Comment (AmyJoT) [inline twitchpopups.htm:117]: Just a small comment - this section could maybe be tabbed in for clarity :)

File: twitchpopups.htm (hunk 7 above)
Comment (mattgirdler) [inline twitchpopups.htm:128]: If you change above to template literals, apply them here too.

File: twitchpopups.htm (hunk 5 above)
Comment (cohan) [reply twitchpopups.htm:127]: Not a part of these commits. Please feel free to fork the repo and fix it.

File: twitchpopups.htm (hunk 6 above)
Comment (cohan) [reply twitchpopups.htm:121]: Beyond the scope of my changes. I was setting up configuration variables, not refactoring.

**Pull request merged**